src/applications/personalization/view-dependents/tests/components/ViewDependentsListItem.unit.spec.jsx

```diff
@@ -27,9 +27,9 @@ describe('<ViewDependentsListItem />', () => {
     );
     expect(
       wrapper.contains(
-        <p className="vads-u-font-weight--bold vads-u-font-size--lg vads-u-text-align--right vads-u-margin--0">
-          Spouse
-        </p>,
+        <dfn className="vads-u-margin--0">
+          <strong>Relationship:</strong> Spouse
```

Comment:
Are we using `<strong>` here more for styling or to convey that the word "Relationship" is of strong importance? 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#Usage_notes